src/tsickle.ts

```diff
@@ -761,6 +764,53 @@ class Annotator extends ClosureRewriter {
     return false;
   }
 
+  private handleSourceFile(sf: ts.SourceFile) {
+    const comments = ts.getLeadingCommentRanges(sf.getFullText(), 0);
+    // Add a @suppress {checkTypes} tag to each source file's JSDoc comment.
+    // Be careful to retain an existing comment and its @suppress'ions
+    let fileoverviewTags: jsdoc.Tag[]|null = null;
+    if (comments && comments.length) {
+      // Find any existing @fileoverview comment and merge our @suppress tag with it.
+      // JSCompiler considers the last comment with @fileoverview (or @externs or @nocompile)
+      // that has not been attached to some other tree node to be the file overview comment.
+      // AJD considers *any* comment mentioning @fileoverview.
+      const parseResult = jsdoc.parse(sf.getFullText().substring(comments[0].pos, comments[0].end));
+      if (parseResult && parseResult.tags.some(t => t.tagName === 'fileoverview')) {
+        fileoverviewTags = parseResult.tags;
+        // Do not emit the fileoverview comment again later, otherwise we'll end up with duplicated
+        // @fileoverview, @modName etc comments.
+        this.skipUpToOffset(comments[0].end);
+      }
+    }
+    if (fileoverviewTags) {
+      // Add @suppress {checkTypes}, or add to the list in an existing @suppress tag.
```

Comment:
does it matter if there's an existing one? doesn't it also work to
```
/**
 * @fileoverview file is nice
 * @suppress{problemA,problemB}
 * @suppress{problemC}
 */
```

If so maybe the complexity here isn't worth the trouble